### PR TITLE
whitespace and typo

### PR DESCRIPTION
**Pull request by rogergzou**

### CODE REVIEW

File: test/Inputs/comment_to_something_conversion.swift [hunk 1]
```diff
@@ -94,15 +94,15 @@ enum A012_AttachToEntities {
   func f1() {}
 // CHECK: {{.*}}DocCommentAsXML=[<Function file="{{.*}}" line="{{.*}}" column="{{.*}}"><Name>f1()</Name><USR>s:FC14swift_ide_test5Brief2f1FT_T_</USR><Declaration>func f1()</Declaration><Abstract><Para>Aaa.</Para></Abstract><Discussion><Para>Bbb.</Para></Discussion></Function>]
 
-  ///Aaa.
+  /// Aaa.
   ///
-  ///> Bbb.
+  /// Bbb.
```
Comment (psophis) [inline test/Inputs/comment_to_something_conversion.swift]: I belive these are test comments for the parser. Changes in this file should be reverted.

**Comment by lattner** [comment]
You're right, thanks.  This broke a regression test too.  I reverted that change in 6026b6d.

-Chris

**Comment by psophis** [comment]
There are two more changes in that file that aren't reverted in 6026b6d

As shown here:
https://github.com/apple/swift/commit/c8b942ba245fc5244f8d460430063d1bc54091a5

**Comment by lattner** [comment]
Understood, are those changes causing a problem?

**Comment by psophis** [comment]
Not that I've found. Just pointing them out in case they were test cases that now don't exist.

**Pull request merged**